resfulservice/src/controllers/curationController.js

```diff
@@ -428,3 +430,45 @@ const createBaseObject = (BaseObject, storedObject) => {
   }
   return curatedBaseObject;
 };
+
+const _getDoiInfo = async (req) => {
+  const { logger, params } = req;
+  logger.info('_getDoiInfo(): Function entry');
+  const { doi } = params;
+  const url = `${doiApi}${doi}`;
+  const headers = {
+    accept: 'application/json'
+  };
+
+  const preparedRequest = {
+    method: 'GET',
+    url,
+    headers
+  };
+  return await axios(preparedRequest);
+};
+
+exports.getDoiInfo = async (req, res, next) => {
+  const log = req.logger;
+  log.info('getDoiInfo(): Function entry');
+  try {
+    const response = await _getDoiInfo(req);
+    successWriter(req, { message: 'success' }, 'getDoiInfo');
+    res.set({
+      'Content-Type': response.headers['content-type'],
+      'Content-Length': response.data.length
+    });
```

Comment:
Is there a reason why we are doing this?

Reply:
If I remember correctly, I was getting some weird errors from the response. I'll check over it again to remember what it was

Reply:
removed in latest commit